airflow/www/views.py

```diff
@@ -528,8 +528,9 @@ def code(self, session=None):
                 code = f.read()
             html_code = highlight(
                 code, lexers.PythonLexer(), HtmlFormatter(linenos=True))
-        except OSError as e:
-            html_code = str(e)
+        except Exception as e:
```

Comment:
This is a very broad exception. Why the change?

Reply:
Because with the test I got AttributeError None object doe not have dag_id.
This was me trying to patch it up.